dallinger/mturk.py

```diff
@@ -290,58 +364,93 @@ def create_hit(self, title, description, keywords, reward, duration_hours,
         self.set_rest_notification(notification_url, hit_type_id)
 
         params = {
-            'hit_type': hit_type_id,
-            'question': mturk_question,
-            'lifetime': datetime.timedelta(days=lifetime_days),
-            'max_assignments': max_assignments,
-            'duration': datetime.timedelta(hours=duration_hours),
-            'approval_delay': None,
-            'response_groups': [
-                'Minimal',
-                'HITDetail',
-                'HITQuestion',
-                'HITAssignmentSummary'
-            ]
+            'HITTypeId': hit_type_id,
+            'Question': mturk_question,
+            'LifetimeInSeconds': datetime.timedelta(days=lifetime_days).seconds,
+            'MaxAssignments': max_assignments,
+            'UniqueRequestToken': self._request_token()
         }
```

Comment:
No duration any more?

Reply:
`LifetimeInSeconds` is I think what you're looking for?

Reply:
The old code passes both lifetime and duration and the new code only passes lifetime, and I'm not sure I understand their semantics.